### DISCUSSION TITLE
Unification of in-app icon colors

### DISCUSSION THREAD

**Discussion by pome-ta** [community/community#3821] [Mobile]
Thanks for watching!

i used this app in dark mode.

home -> repository icon[gray]
profile -> repository icon[purple]

Unless there is a special reason, I would be happy if we could unify the colors.

When making a momentary decision, I find it difficult because the colors are different.

**Comment by nplasterer** [comment]
Thanks for the feedback! I've passed this along to the team to look into how we can make this better.
